src/test/mockLocalMain.ts

```diff
@@ -0,0 +1,40 @@
+import 'jest-extended';
```

Comment:
I believe this only needs to be imported if we specifically need access to one or more of its assertion helpers. Did you find that you need this imported in this file specifically?

Reply:
I think this is leftover from something else I was trying. Deleting this!